maven-java/kalix-spring-boot-kotlin-archetype/src/main/resources/archetype-resources/README.md

```diff
@@ -0,0 +1,56 @@
+# ${artifactId}
+
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project contains the framework to create a Kalix application by adding Kalix components. To understand more about these components, see [Developing services](https://docs.kalix.io/services/). Spring-SDK is an experimental feature and so far there is no [official](https://docs.kalix.io/) documentation. Examples can be found [here](https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples) in the folders with "spring" in their name.
```

Comment:
Not so experimental anymore and we actually have docs \o/

Reply:
Hmm, we should adapt the spring boot java archetype as well.